pkgs/by-name/co/composefs/package.nix

```
@@ -69,7 +70,12 @@ stdenv.mkDerivation (finalAttrs: {
 
   passthru = {
     updateScript = nix-update-script { };
-    tests.pkg-config = testers.testMetaPkgConfig finalAttrs.finalPackage;
+    tests = {
+      pkg-config = testers.testMetaPkgConfig finalAttrs.finalPackage;
+    } // lib.optionalAttrs (stdenv.hostPlatform.system != "aarch64-linux") {
```

```suggestion
    } // lib.optionalAttrs (stdenv.hostPlatform.system == "x86_64-linux") {
```
we only know which platforms work, so we should filter by that

---

Also this just filters out the tests from this package, not actually disabling the test on x86_64-linux. The correct way would be to disable the test like https://github.com/NixOS/nixpkgs/blob/master/nixos/tests/all-tests.nix#L271 but since we likely want to keep the test on all platforms I think we can probably remove this condition again and just note for the time being that it fails on aarch64-linux and that is unrelated

/cc @nikstur

---

reverted the change and added a note